### PR TITLE
inserted side bar from ant design, modified options to match figma prototype

### PR DESCRIPTION
**Pull request by andrewskr90**
What I did:
Imported instructor sidebar icons and added Ant-D sider component to InstructorSidebar.js. Incorporated icons and titles to sidebar, wrote onCollapse function, and applied onCollapse to collapse aspect of sider

Files changed:
src/components/pages/InstructorHome/InstructorSidebar.js

Why were they changed:
In order to start building out the UI for the instructor's dashboard. The sidebar acts as the foundation to the instructor page's layout

Coauthored by @Jie-chelchel 
!(https://www.loom.com/share/126da2346a654a6e8c6dd2d1fe267fec)

### CODE REVIEW

**Review by cn8817 — APPROVED** [review]
Looks great, very clean code!

**Review by muhannadbm — APPROVED** [review]
Nice work, i think replacing the int key with strings is better, Good job!

File: src/components/pages/InstructorHome/InstructorSidebar.js [hunk 1]
```diff
@@ -1,43 +1,52 @@
...
+      setCollapsed(false);
+    } else {
+      setCollapsed(true);
+    }
+  };
+
   return (
     <Layout style={{ minHeight: '100vh' }}>
-      <Sider>
+      <Sider collapsible collapsed={collapsed} onCollapse={onCollapse}>
```
Comment (JoeyMBrown) [inline src/components/pages/InstructorHome/InstructorSidebar.js:30]: This collapsing looks AMAZING in your loom video - super blown away, nice job! @andrewskr90 @Jie-chelchel

**Review by JoeyMBrown — APPROVED** [review]
Nice work!

**Pull request merged**